app/screens/interactive_dialog/index.js

```diff
@@ -20,6 +21,7 @@ function mapStateToProps(state) {
         elements: data.dialog.elements,
         title: data.dialog.title,
         iconUrl: data.dialog.icon_url,
+        isLandscape: isLandscape(state),
```

Comment:
consider passing this value from the parent component, and if not available consider calculating it by using `Dimensions.get('window')`.

The reason behind it is to make it simpler when migrating to realm as the state for landscape won't be available anymore.

Reply:
Okay, did the latter by computing from `Dimensions`.  Also add listener for dimension/orientation change.